pandas/tests/io/test_gbq.py

```diff
@@ -147,25 +147,25 @@ def mock_read_gbq(sql, **kwargs):
 
 @pytest.mark.single
 class TestToGBQIntegrationWithServiceAccountKeyPath:
+    @staticmethod
+    def generate_rand_str(length: int = 10) -> str:
+        return "".join(random.choices(string.ascii_lowercase, k=length))
+
     @pytest.fixture()
     def gbq_dataset(self):
         # Setup Dataset
         _skip_if_no_project_id()
         _skip_if_no_private_key_path()
 
-        dataset_id = "pydata_pandas_bq_testing_py31"
+        dataset_id = "pydata_pandas_bq_testing_" + self.generate_rand_str()
 
         self.client = _get_client()
         self.dataset = self.client.dataset(dataset_id)
-        try:
```

Comment:
hmm, are we supposed to clean up these datasets? @tswast

Reply:
We now just do this here:
https://github.com/pandas-dev/pandas/blob/a91b8a3dfe0a3c6efea4a33a06676ad1ea63055e/pandas/tests/io/test_gbq.py#L173